### PR TITLE
Add EFD logMessage endpoint

### PR DESCRIPTION
**Pull request by sebastian-aranda**
This PR adds an improvement feature that will allow the EventLog component to query the EFD. A new commander endpoint was added.

### CODE REVIEW

File: commander/efd.py [hunk 1]
```diff
@@ -90,16 +91,84 @@ async def query_efd_timeseries(request):
...
+            cscs = req["cscs"]
+        except Exception:
+            return web.json_response(
+                {"ack": "Some of the required parameters is not present"}, status=400
+            )
+
+        efd_client = efd_clients.get(efd_instance)
+        if efd_client is None:
+            efd_client = connect_to_efd_intance(efd_instance)
+        if efd_client is None:
```
Comment (areyesd14) [inline commander/efd.py:111]: Is it correct that this "if" is repeated and with two different results? In the first "if" you assign a value to "efd_client" and in the second you return a function. Could it be that the first if is "if efd_client is not None"?

File: commander/efd.py [hunk 2]
```diff
@@ -90,16 +91,84 @@ async def query_efd_timeseries(request):
...
+            return unavailable_efd_client()
+
+        start_date = Time(start_date, scale="utc")
+        end_date = Time(end_date, scale="utc")
+        query_tasks = []
+        sources = []
+        for csc in cscs:
+            indices = cscs[csc]
```
Comment (areyesd14) [inline commander/efd.py:119]: Maybe keep the name of the variables in English?

**Review by areyesd14 — APPROVED** [review]
I left some comments, but the rest looks good. Thanks @sebastian-aranda !

File: commander/efd.py (hunk 1 above)
Comment (sebastian-aranda) [reply commander/efd.py:111]: Yes, this is correct 😉️. This works as follows:
1. Check if we have a connected instance saved at `efd_clients`
2. If there is no connected instance (`efd_client is None == True`), then try to connect to the instance
3. If on the previous step the instance could not be connected, then we return an error response with status 400

Comment (sebastian-aranda) [reply commander/efd.py:111]: Instances will be connected on demand, this is why we use two different if statements. We could nest the second if on the first one, but in this way it also works

File: commander/efd.py (hunk 2 above)
Comment (sebastian-aranda) [reply commander/efd.py:119]: This is already in English jeje

**Pull request merged**